drivers/i2c/i2c_esp32.c

```diff
@@ -284,6 +284,19 @@ static int i2c_esp32_configure(struct device *dev, u32_t dev_config)
 	return ret;
 }
 
+static void i2c_esp32_reset_cmd(const struct i2c_esp32_config *config)
```

Comment:
Could this be changed so that, right before a transfer is performed, only the unused commands are cleared out? This way, the number of unneeded register writes is reduced, and command queue cleanup happens in only one place.

Reply:
Done.

Reply:
Actually this cleanup is unnecessary because of the way commands are inserted in the queue, I'm going to remove this commit.